.gitignore

```diff
@@ -4,6 +4,7 @@ todo.txt
 *.sqlite
 
 .cache/
+cache/
```

Comment:
I also added `cache/` as an ignore'd directory as a convenience. (IIRC, `.cache` was also used specifically for an in-library HTTP cache, not user-directed caching.)